edx_solutions_api_integration/mobile_api/urls.py

```diff
@@ -15,4 +15,10 @@
         name='mobile-courses-overview'),
     url(r'^users/courses/{0}'.format(COURSE_ID_PATTERN), mobile_views.MobileUsersCoursesDetail.as_view(),
         name='mobile-users-courses-detail'),
+    url(r'^courses/{0}/static_tabs/*$'.format(COURSE_ID_PATTERN), mobile_views.MobileCoursesStaticTabsList.as_view(),
+        name='mobile-courses-static-tabs-list'),
+    url(r'^courses/{0}/static_tabs/(?P<tab_id>[a-zA-Z0-9_+\/:-]+)$'.format(COURSE_ID_PATTERN),
+        mobile_views.MobileCoursesStaticTabsDetail.as_view()),
+    url(r'^courses/{0}/static_tabs/(?P<tab_id>[a-zA-Z0-9_+\s\/:-]+)$'.format(COURSE_ID_PATTERN),
```

Comment:
I think we should fix this url pattern by finding a way to combine these regexes into one.